### ISSUE TITLE
Flow resolvers - Missing default generics annotation for generated interface types

### ISSUE THREAD

**Issue by emmenko** [dotansimha/graphql-code-generator#1650]
**Describe the bug**
The generated types for interface types are missing a default generics annotation `<>`, causing flow to throw an error:

<img width="706" alt="image" src="https://user-images.githubusercontent.com/1110551/55537836-5e280700-56bd-11e9-9489-08de8f3ba098.png">

**To Reproduce**

1. My GraphQL schema:

```graphql
interface MetaData {
  id: ID!
  # ...
};
```

2. My config file:

```js
const gqlConfig = {
  overwrite: true,
  schema: `http://localhost:${port}/graphql`,
  generates: {
    'lib/types/generated/graphql-types.js': {
      plugins: [`flow`],
    },
    'lib/types/generated/graphql-resolvers.js': {
      plugins: [
        { add: '/* @flow */' },
        { add: "import type { GraphQLContext } from '../graphql';" },
        `flow`,
        `flow-resolvers`,
      ],
      config: {
        contextType: 'GraphQLContext',
      },
    },
  },
};
```

**Expected types generated**

```js
export type Resolvers<Context = GraphQLContext> = {
  MetaData?: MetaDataResolvers<>,
};
```

**Actual types generated**

```js
export type Resolvers<Context = GraphQLContext> = {
  MetaData?: MetaDataResolvers,
};
```

**Comment by dotansimha** [comment]
Fixed in: https://github.com/dotansimha/graphql-code-generator/pull/1663 
@emmenko , can you please try the latest alpha from that PR? (`1.0.7-alpha-b4949752.51`)

**Comment by dotansimha** [comment]
Fixed in 1.0.7 :)
